Hector101/e-Document-Manager
# chore(integrate-test-runners):

#### What does this PR do?
scaffold the project folder structure, integrate Travis, HoundCI, Code Climate, setup node server and integrate test.

#### Description of Task to be completed?
Integrate Travis, HoundCI, Code Climate, define database model, setup node server

#### How should this be manually tested?
create a PR and check if travis or hound ci builds to check if your code is following best practice

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
